cli/cmd/scan/scan.go

```diff
@@ -179,6 +179,9 @@ func loadConfig(cfgFile string) *scanner.Config {
 	err := viper.ReadInConfig()
 	cobra.CheckErr(err)
 
+	// Set defaults
+	viper.SetDefault("malware.scanners_config.clam.use_clam_daemon", true)
```

Comment:
we can replace `use_clam_daemon` with `use_native_clamscan`. its not a breaking change yet since we still havent released. this would default to using daemon for scanning. 

use another flag that defaults to daemon scan if you have a better suggestion, but we can definitely remove this part to avoid creating partial defaults scattered around the code base

Reply:
Indeed, now I see in the history this is a new config option. Replaced use_clam_daemon with use_native_clamscan.